R/species.R

```diff
@@ -0,0 +1,48 @@
+#' Get data from bwg
+#' 
+#' At present this is very general, since perhaps we will 
+#' only ever get whole tables?
+#' 
+#' @param dataname the name of the dataset you want (e.g.
+#'   "species")
+#'   
+#' @return a data.frame containing one whole table.
+#' @export
+bwg_get <- function(dataname) {
+  
+  ## first, check if there is a token
+  assertthat::assert_that(exists("token", envir = parent.frame()))
+  
+  token <- get("token", envir = parent.frame())
```

Comment:
Look, all that time reading advanced R paid off! Except I fear this will cause problems. Perhaps if people are in the habit of saving their R workspaces..